packages/frint-cli/commands/new.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-multi-spaces */
+
+const mkdirp = require('mkdirp');
+const request = require('request');
+const tar = require('tar');
+
+const createApp = require('frint').createApp;
+
+const descriptionText = `
+Usage:
+
+  $ frint new
+  $ frint new <directory>
+  $ frint new <directory> --example <example>
+
+Example:
+
+  $ frint new myapp --example kitchensink
+  $ frint new myapp --example frint-vue/tree/master/examples/basic
+
+You can find a list of all available official examples here:
+https://github.com/Travix-International/frint/tree/master/examples
+`.trim();
+
+const invalidExampleArgText = `
+Invalid <example> value. Must be in one of the following formats:
+
+  * <name>
+  * <organization>/<repository>/tree/<branch>/**/<name>
+`.trim();
+
+module.exports = createApp({
+  name: 'new',
+  providers: [
+    {
+      name: 'summary',
+      useValue: 'Scaffolds a new Frint app in specified directory',
+    },
+    {
+      name: 'description',
+      useValue: descriptionText,
+    },
+    {
+      name: 'execute',
+      useFactory: function useFactory(deps) {
+        return function execute() {
+          // The <example> param has two shapes:
+          // * <name> - example name from the official Frint GitHub repository
+          // * <organization>/<repo>/tree/<branch>/** - full GitHub path to an arbitrary example
+          const example = deps.params.example || 'counter';
+
+          // Split by '/' and filter out empty results.
+          // <example> arg might start or end with a separator.
+          const exampleParts = example.split('/').filter(str => str !== '');
```

Comment:
is this check whitespace proof?

Reply:
Nope, but such a value is currently considered invalid format with undefined behavior. I could add additional regex validation if that helps.

Reply:
I would extract this as a separate function:

```js
function getDownloadUrlFromExample(example) {
  if (example.indexOf('/') === -1) {
    return `https://.../tar.gz/${DEFAULT_BRANCH}`;
  }

  return customUrl;
}
```